docs/topics/development/testing.rst

```diff
@@ -178,9 +178,9 @@ need to recompile the .mo files manually, for example::
 
 .. _`Django's Unit Testing`: http://docs.djangoproject.com/en/dev/topics/testing
 .. _`PyPom`: http://pypom.readthedocs.io/en/latest/
-.. _`pytest`: http://pytest.org/latest/
+.. _`pytest`: http://pytest.org/en/latest/
```

Comment:
Let's use only `pytest.org` here instead to help the user land on a page in his personal preferred language.

Reply:
Thanks @EnTeQuAk I updated this PR.